packages/libs/pactjs/src/PactExpression.ts

```diff
@@ -0,0 +1,33 @@
+import { parseType } from './utils/parseType';
+
+export interface IPactModules {}
+export type IPactBuilder = {
+  generate(): string;
+} & IPactModules;
+
+export function PactExpression(): IPactBuilder {
+  let expression: string = '';
+  const path: string[] = [];
+  const args: (string | number)[] = [];
+  const builder: IPactBuilder = new Proxy(
+    function () {} as unknown as IPactBuilder,
+    {
+      get(target: unknown, prop: string, receiver: unknown) {
+        if (prop === 'generate') {
+          expression = `(${path.join('.')}${args.length > 0 ? ' ' : ''}${args
+            .map(parseType)
+            .join(' ')})`;
+          return () => expression;
+        }
+        path.push(prop);
+        return builder;
+      },
```

Comment:
Same as last comment, `createExp`. Also, there can be nested pact codes, like `(func (read-keyset 'hi ))`, so we should add pact code as input type? 
 There also needs to be separation between strings and pact values, i.e.) `(func some-val "arg")` where we don't want some-val to be wrapped in quotations but we want `arg` to be stringified.  - I think something to add in pactjs

Reply:
Yes. I'm now using `parseType` to "stringify" to parse the arguments based on their inputs, but that doesn't fix the type for `some-value` vs `"some-string"`. The generator should probably be aware of the type that's been passed. Maybe we need to create PactString, PactValue, PactSymbol, etc.